app/connectors/PSRConnector.scala

```diff
@@ -45,15 +45,15 @@ class PSRConnector @Inject()(appConfig: FrontendAppConfig, http: HttpClient) {
 
   def submitPsrDetails(
     psrSubmission: PsrSubmission
-  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[String, Boolean]] =
+  )(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[String, Unit]] =
     http
       .POST[PsrSubmission, HttpResponse](
         submitStandardUrl,
         psrSubmission
       )
       .map { response =>
         response.status match {
-          case NO_CONTENT => Right(true)
+          case NO_CONTENT => Right(Future.unit)
```

Comment:
why Future.unit ? 
this is `final val unit: Future[Unit] = fromTry(Success(()))`

Reply:
I thought they are equivalent. But I can change it, no problems.